devtools/common/src/main/resources/templates/dockerignore.ftl

```diff
@@ -1,2 +1,2 @@
 *
-!target/${project_artifactId}
\ No newline at end of file
+!target/*-runner
```

Comment:
@geoand does it make sense to to -runner and -runner.jar

Reply:
@cescoffier That is what I am doing in #1312 :wink: . Would you like me to add it in this PR and get it over with?

Reply:
Yes, let's do it once for all.

Reply:
Makes sense! Fixed